src/rules.cpp

```diff
@@ -10,6 +10,8 @@
 #include "utils.h"
 #include "xconnection.h"
 
+using std::regex;
+using std::regex_error;
 using std::string;
```

Comment:
Why `using std::regex`? I'd prefer not `use` it globally but to keep the `std::` prefix here..

Reply:
Ah, it's my mistake: there is:
```
regexstr.cpp:using std::regex;
```
So it's in the scope of another PR